arrow-fx-coroutines/src/test/kotlin/arrow/fx/coroutines/stream/Arb.kt

```diff
@@ -43,16 +42,21 @@ inline fun <reified A> Arb.Companion.array(
   }
 }
 
-fun <A, B> arrayChunkGenerator(
-  gen: Gen<A>,
+@PublishedApi
+internal fun <A, B> arrayChunkGenerator(
```

Comment:
Right, I can move these to a separate PR. No problem. Should we figure out some guidelines?

Reply:
I'm not sure what @aballano is used to exactly, but on my side I'm used to [Gitflow](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow) or similar variants of it. However, that's something to be agreed with everybody, I'd not block this PR for a decision like that. These flows also cover things like release branches that we could use for our docs thing discussed yesterday.

That said, the single responsibility for PRs applies the same way for any workflows, since it's more like a matter of making those as atomic as possible, simply to make it easier for reviewers. Imho, ideally every PR would cover a single topic, and be as atomic as you can for segregation. (Almost any PR can be chunked on very differentiated steps of a whole).

That said also, If a "cowboy" change is quite related to the changes on the diff, then I guess it's fine, but there is always a point where you start feeling like it's too much to include. At that point it's probably better as a separate PR with its proper description / discussion. I know common sense is not a very common sense, but this would be more on every one to consider and evaluate.

Reply:
> I know common sense is not a very common sense, but this would be more on every one to consider and evaluate.

"common sense" :clap: and I'd add "empathy": _could someone review this PR easily?_

Reply:
IMO changes like this in tests for small PRs like this one shouldn't matter (at least to me personally) as long as this doesn't turn into the main changes of the PR